.github/workflows/tests.yml

```diff
@@ -41,29 +41,59 @@ env:
   # GITHUB_CONTEXT: ${{ toJson(github) }}
 
 jobs:
+  check:
```

Comment:
Added this to create a one-stop-shop for the rules around what should or should not run.  Should make maintenance easier in future.